apps/studio/components/to-be-cleaned/Storage/StorageExplorer/FileExplorerRow.tsx

```diff
@@ -113,25 +114,23 @@ const FileExplorerRow: ItemRenderer<StorageItem, FileExplorerRowProps> = ({
   openedFolders = [],
 }) => {
   const { ref: projectRef, bucketId } = useParams()
-  const storageExplorerStore = useStorageStore()
+  const snap = useStorageExplorerStateSnapshot()
```

Comment:
Duplicate usage of the `useStorageExplorerStateSnapshot`?

Reply:
oh yes good catch! will fix 🙏